superset-frontend/src/components/IconTooltip.tsx

```diff
@@ -69,11 +74,11 @@ const Link = ({
         mouseEnterDelay={0.3}
         mouseLeaveDelay={0.15}
       >
-        {link}
+        {iconTooltip}
       </Tooltip>
     );
   }
-  return link;
+  return iconTooltip;
 };
 
-export default Link;
+export default IconTooltip;
```

Comment:
personally, I prefer named exports to better encourage uniform naming/usage on the import side... but this isn't your fault, and I'm not sure if it's worth changing here.

Reply:
I also prefer named exports lol. I kept because of the current code style but I think we can use the same approach as typescript and use named exports for new components. Since this is a new component I'll make the change =D